RecCalorimeter/src/components/CreateCaloCells.cpp

```diff
@@ -84,13 +90,41 @@ StatusCode CreateCaloCells::execute() {
     m_cellsMap.clear();
   }
 
+  m_HitCellsMap.clear();
```

Comment:
Can you make `m_HitCellsMap` just a local variable? This way you will not have to clear it in every event.

Reply:
I changed the name of the variable to "HitCellsMap" and made it local